### PR TITLE
Quick guide go1.18

### PR DESCRIPTION
**Pull request by cuua**
请review

### CODE REVIEW

File: 2021/w48_quick_guide_go1.18_features.md [hunk 1]
```diff
@@ -5,32 +5,35 @@
...
 func minFloat(x float32, y float32) float32 {
    if x < y {
       return x
    } else {
       return y
    }
 }
-With the power of Type Parameters, we could easily define a function that could get desired types of variables. For defining types, we also encounter a new operator called tilde ~. This operator actually returns us the interface of the type information, so we can make our type restrictions. Let's write the same min function that covers int and float32 types as below.
+```
+通过泛型的能力，我们可以轻松的定义一个函数，这样我们可以得到所需类型的变量。通过定义类型，我们还可以看到一个名为~的新操作符。这个操作符实际上返回给我们的是类型信息的接口，因此我们可以进行类型限制，让我们来编写相同的min函数来覆盖int和float32，如下所示：
```
Comment (zxmfke) [inline 2021/w48_quick_guide_go1.18_features.md:34]: cover也可以翻译为包含，想要表达的意思应该是这个min函数修改成可以包含int和float32的传参

File: 2021/w48_quick_guide_go1.18_features.md [hunk 2]
```diff
@@ -41,25 +44,28 @@ func min[P OrderTypes](x, y P) P {
...
     if x < y {
       return x
     } else {
       return y
     }
 }
-Of course, the use of generics is not limited to this sample. It could be very useful especially in ORM-style problems that we generally create functions that work with multiple types. As well, you could check these examples shared by the Go team also give you some ideas. If you desire to watch the details, there is a great presentation the team made at GopherCon.
-Apart from these, the online editor (playground) where we can experiment will also be useful. If you wish to try 1.18 in your own environment, you could download gotip and work with the active branch of Go.
+```
+当然，泛型的使用不仅仅限于这个例子。它可能在ORM风格的问题中特别有用，我们通常创建使用多种类型的函数，此外你还可以看到Go团队分享的示例。如果你想看细节，团队在GopherCon做了一个很棒的演示。
```
Comment (zxmfke) [inline 2021/w48_quick_guide_go1.18_features.md:58]: 它会非常有用，尤其ORM-style类型的问题中，在那种我们创建需要处理多个类型参数的函数。同时，你还可以跟着过一下GO团队分享的例子，从中得到一些想法。

File: 2021/w48_quick_guide_go1.18_features.md [hunk 3]
```diff
@@ -41,25 +44,28 @@ func min[P OrderTypes](x, y P) P {
...
-Fuzzing Test
-With the Golang 1.18 package, the “Fuzzing” feature will come into our lives. With this test library, which will be offered as beta, we will be able to automatically make random mutations of our inputs in unit tests.
+除此之外，我们可以进行实验的在线编辑（playground）也很有用。如果你希望使用1.18在你自己的环境，你可以下载gotip并使用Go的活跃的分支。
 
-As software developers, we sometimes miss unit tests edge cases. And this lack of data diversity could be abused and used in security leaks and vulnerabilities especially on critical libraries. We could prevent this situation using Fuzzing Test.
+#模糊测试
+随着Go1.18软件包的发布，"模糊化"功能将进入我们的生活。有了这个在beta版提供的测试库，我们将能够在单元测试中自动做随机突变输入。
 
-For implementing Fuzzing Test, you could wrap your unit tests with a method that name starts with the Fuzz prefix. You could also use the Go Testing page as a sample code.
+作为软件开发者，我们有时会忘记边缘测试。这种数据多样性的缺乏可能会被滥用，导致安全漏洞，特别是在关键库上。我们可以通过模糊测试来预防这种情况。
```
Comment (zxmfke) [inline 2021/w48_quick_guide_go1.18_features.md:65]: 我们有时会忘了单元测试的边缘案例。特别在重要的库上，这种缺少数据的多样性可能会被滥用，也会导致安全泄漏和漏洞。

File: 2021/w48_quick_guide_go1.18_features.md [hunk 4]
```diff
@@ -75,11 +81,13 @@ func FuzzHex(f *testing.F) {
     }
   })
 }
-Workspaces
-This feature comes to facilitates developers' jobs, who work on multiple modules at the same time. Although the module version can be changed with the replace command in go.mod version 1.17, it is often forgotten to clear these replace commands while committing the code. Inserting or removing that line every time is a very bothering thing to do for these developers.
+```
+#工作区
+此功能有助于开发人员同时处理多个模块。尽管可以在go.mod 1.17版本中使用replace命令更改module版本。但是在提交代码的时候，常常忘记清除这些replace命令。对于我们开发人员来说，每次插入或删除该行都是一件非常麻烦的事情。
```
Comment (zxmfke) [inline 2021/w48_quick_guide_go1.18_features.md:86]: 此功能提升了同时需要处理多个模块的开发人员的便利性
